src/cartservice/Program.cs

```diff
@@ -51,8 +54,27 @@ static object StartServer(string host, int port, ICartStore cartStore)
             {
                 try
                 {
+
                     await cartStore.InitializeAsync();
 
+                    var redisCartStore = cartStore as RedisCartStore;
+
+                    if (redisCartStore != null)
+                    {
+                            tracerProvider = OpenTelemetry.Sdk.CreateTracerProviderBuilder()
```

Comment:
Were you planning on configuring OTel with a resource? If so, something like

```csharp
using OpenTelemetry.Resources;

...

tracerProvider = OpenTelemetry.Sdk.CreateTracerProviderBuilder()
    .SetResource(CreateServiceResource("CartService"));
```

Then you can remove setting of `options.ServiceName` below for our exporter config.

Reply:
oops, yeah it was in my local branch 😬